### ISSUE TITLE
Field of file type are being reset to single line text when trying to import previously created forms

### ISSUE THREAD

**Issue by swapnilmanjrekar** [CalderaWP/Caldera-Forms#2859]
## Is This A Feature Request?
* No

## What Version Of Caldera Forms, WordPress and PHP Are You Using?
* WordPress Version: 5.0.1PHP Version: 7.1.25MySQL Version: 5.7.24Caldera Forms Version: 1.7.4WP_DEBUG: 1

## Does Your Issue Persist When You Disable All Other Plugins and Switch To The Default Theme?
* Yes

## What Is The Unexpected Behaviour?
*  When I try to import my previously created forms in WordPress 4.9 to 5.0.1, the file fields of my form are being reset from type file field to a single line text type field.

## What PHP Errors Have You Logged While Reproducing This Bug?
* PHP Warning:  array_key_exists() expects parameter 2 to be array, null given in /app/wp-content/plugins/caldera-forms/classes/forms.php on line 274

## What JavaScript Errors Have You Seen While Reproducing This Bug?
* None

**Comment by Shelob9** [comment]
@swapnilmanjrekar Good catch.

I think the issue here is that that a adv_file no longer exists, so it falls back to text field. We need to adjust file type in the import process.

**Comment by swapnilmanjrekar** [comment]
@Shelob9 I was not able to reproduce the issue as I was able to import my previously created file field form and after Importing the form it was properly displaying File field in the form. Also, I did not see any JS or PHP errors when I checked the console of my browsers ie. Chrome, Edge & Firefox and debug.log file of my local WordPress test site.

WordPress Version: 5.0.3PHP Version: 7.1.26MySQL Version: 5.7.25Caldera Forms Version: 1.8.0-beta.1WP_DEBUG: 1
